2000-08-02  Bernd Schmidt  <bernds@cygnus.co.uk>

	* loop.c (maybe_eliminate_biv_1): Turn off yet another case.

--- gcc/loop.c.baseline	Sat Jul 29 14:38:22 2000
+++ gcc/loop.c	Wed Aug  2 17:21:44 2000
@@ -8625,7 +8622,8 @@
 	{
 	  /* First try to replace with any giv that has constant positive
 	     mult_val and constant add_val.  We might be able to support
-	     negative mult_val, but it seems complex to do it in general.  */
+	     negative mult_val, but it seems complex to do it in general.
+	     ??? Turn this off due to possible overflow.  */
 
 	  for (v = bl->giv; v; v = v->next_iv)
 	    if (GET_CODE (v->mult_val) == CONST_INT && INTVAL (v->mult_val) > 0
@@ -8635,7 +8633,8 @@
 		    || (GET_CODE (v->add_val) == REG
 			&& REGNO_POINTER_FLAG (REGNO (v->add_val))))
 		&& ! v->ignore && ! v->maybe_dead && v->always_computable
-		&& v->mode == mode)
+		&& v->mode == mode
+		&& 0)
 	      {
 		if (! biv_elimination_giv_has_0_offset (bl->biv, v, insn))
 		  continue;

